sympy/solvers/ode/tests/test_single.py

```diff
@@ -1129,6 +1133,138 @@ def _get_examples_ode_sol_nth_linear_undetermined_coefficients():
     }
 
 
+def _get_examples_ode_sol_separable():
+    # test_separable1-5 are from Ordinary Differential Equations, Tenenbaum and
+    # Pollard, pg. 55
+    a = Symbol('a')
+    return {
+            'hint': "separable",
+            'func': f(x),
+            'examples':{
+    'separable_01': {
+        'eq': f(x).diff(x) - f(x),
+        'sol': [Eq(f(x), C1*exp(x))],
+    },
+
+    'separable_02': {
+        'eq': x*f(x).diff(x) - f(x),
+        'sol': [Eq(f(x), C1*x)],
+    },
+
+    'separable_03': {
+        'eq': f(x).diff(x) + sin(x),
+        'sol': [Eq(f(x), C1 + cos(x))],
+    },
+
+    'separable_04': {
+        'eq': f(x)**2 + 1 - (x**2 + 1)*f(x).diff(x),
+        'sol': [Eq(f(x), tan(C1 + atan(x)))],
+    },
+
+    'separable_05': {
+        'eq': f(x).diff(x)/tan(x) - f(x) - 2,
+        'sol': [Eq(f(x), C1/cos(x) - 2)],
+    },
+
+    'separable_06': {
+        'eq': f(x).diff(x) * (1 - sin(f(x))) - 1,
+        'sol': [Eq(-x + f(x) + cos(f(x)), C1)],
+    },
+
+    'separable_07': {
+        'eq': f(x)*x**2*f(x).diff(x) - f(x)**3 - 2*x**2*f(x).diff(x),
+        'sol': [Eq(f(x), (-x + sqrt(x*(4*C1*x + x - 4)))/(2*(C1*x - 1))),
+        Eq(f(x), -(x + sqrt(x*(4*C1*x + x - 4)))/(2*(C1*x - 1)))],
+        'slow': True,
+    },
+
+    'separable_08': {
+        'eq': f(x)**2 - 1 - (2*f(x) + x*f(x))*f(x).diff(x),
+        'sol': [Eq(-log(-1 + f(x)**2)/2, C1 - log(2 + x))],
+        'slow': True,
+    },
+
+    'separable_09': {
+        'eq': x*log(x)*f(x).diff(x) + sqrt(1 + f(x)**2),
+        'sol': [Eq(asinh(f(x)), C1 - log(log(x)))], # integrate cannot handle the integral on the lhs (cos/tan)
```

Comment:
I guess this comment applies to the example below.

Also why in this example has it not solved for `f(x)`?

Reply:
> I guess this comment applies to the example below.
sorry, it is my mistake.  I will fix this.

> Also why in this example has it not solved for `f(x)`?

yup, i was testing for examples which are not marked as slow, for slow examples, i will be doing in my next commit.